scss/components/alert.scss

```diff
@@ -0,0 +1,52 @@
+@import "../core/settings";
+@import "../core/mixins";
+@import "../core/functions";
+
+/*!
+.tn-alert+(--notification. --error)
+    .tn-alert__action
+*/
+$block: ns(alert);
+
+.#{$block} {
+    //SETTINGS
+    $tn-alert-background-color: tn-color(text, 3) !default;
+    $tn-alert-background-color--notification: tn-color(status, 2) !default;
+    $tn-alert-background-color--error: tn-color(status, 3) !default;
+    $tn-alert-color: tn-color(text-inverse, 1) !default;
+    $tn-alert-border-color: tn-color(neutral, 2) !default;
+    $tn-alert-height: tn-space(13) !default;
+
+    // Block
+    color: $tn-alert-color;
+    border: solid 1px $tn-alert-border-color;
+    background-color: $tn-alert-background-color;
+    height: $tn-alert-height;
+    padding: tn-space(s);
+    text-align: center;
+
+    // Element
+    &__action {
+        float: right;
+        margin: -(tn-space(s));
+        .tn-button{
```

Comment:
We want to avoid this ... changing components based on their parent. 

So there are a few options ...

1. Add a modification for buttons so you can get the inverse type without a background - `.tn-button--clear` or `tn-button--inverse`
2.  Add a button mixin where you can embed button styles on the fly into different elements - `.z-button { @include tn-button($color, $background) }`
3. Create a specific button for this component.

Since we only have a single use case thus far for an inverse button I don't recommend #1, same for #2 although we may need this later if we need different buttons with different backgrounds, so #3 is the best based on the info we have at this time.

I recommend creating a `.tn-alert__close` button that has an icon inside. Then the CSS here could be ..

```
    &__action {
        float: right;
        margin: tn-space(2) -(tn-space(s));
    }
    &__close {
        @include tn-button-reset;  //removes default button chrome
        color: tn-color(text-inverse, 1);
        width: tn-space(m); //best practice hit space
        height: tn-space(m);
}
```

The button-reset mixin is part of component mixins. See above for `@import`